packages/owl-bot/test/handlers.ts

```diff
@@ -524,4 +552,18 @@ describe('scanGithubForConfigs', () => {
       ])
     );
   });
+
+  it('recovers from 404 when scanning configs', async () => {
```

Comment:
a repo with no content 404s when `getBranch()` is called.

Reply:
Thanks for adding tests!